sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -2334,8 +2333,7 @@ private void updatePersonAndCaseByOutcome(CaseDataDto existingCase, Case newCase
 			}
 
 			if (newCase.getOutcome() == CaseOutcome.DECEASED) {
-				if (newCase.getPerson().getPresentCondition() != PresentCondition.DEAD
-					&& newCase.getPerson().getPresentCondition() != PresentCondition.BURIED) {
+				if (!newCase.getPerson().getPresentCondition().isDeceased()) {
```

Comment:
Please don't merge yet, I am currently investigating why this causes a potential NullPointerException